packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1090,7 +1092,7 @@ export default class Meetings extends WebexPlugin {
    */
   public create(
     destination: string,
-    type: string = null,
+    type: string | MeetingType = null,
```

Comment:
why is the type still allowed to be string too? and why do we convert it to string later on and still use old set of consts? Changing everywhere to use the new type enum shouldn't be that difficult, it's only mainly used in the fetching meeting info code (as destinationType) and only in a couple of other place that are easy to fix (like MeetingsUtil.getMeetingAddedType). It feels like maybe we should even rename the new enum from `MeetingType` to `DestinationType`